src/bucket.ts

```diff
@@ -244,6 +244,7 @@ export interface GetBucketSignedUrlConfig {
   action: 'list';
   version?: 'v2' | 'v4';
   cname?: string;
+  urlStyle?: 'virtual-host' | 'path';
```

Comment:
I'd like this to be a boolean rather than a string value if possible. 
```
virtualHostedStyle: true | false.
```

By default it's 'path' style and the documentation here can be simplified.